hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/type/PortableNestedFieldsTest.java

```diff
@@ -59,37 +67,69 @@ public static void beforeClass() {
                 .build();
         serializationService.getPortableContext().registerClassDefinition(organizationType);
 
+        organizationAndLongType = new ClassDefinitionBuilder(1, 4)
```

Comment:
why don't we configure these class definitions before the instance starts, through the serialization config?